tests/e2e/expected_results/maven/maven.json

```diff
@@ -212,8 +212,8 @@
         }
     },
     "dependencies": {
-        "analyzed_deps": 0,
-        "unique_dep_repos": 0,
```

Comment:
Why is there a change in the number of dependencies for this analysis target?

Reply:
That's because we use the same `maven.json` expectation file for different e2e tests, but we don't actually check the dependencies in the e2e comparison script.